src/app/services/data/trainrunsection.service.ts

```diff
@@ -988,15 +1000,15 @@ export class TrainrunSectionService implements OnDestroy {
     );
     let travelTime1 =
       trainrunSection1.getTargetDepartureConsecutiveTime() -
-      trainrunSection1.getSourceDepartureConsecutiveTime();
+      trainrunSection1.getSourceArrivalConsecutiveTime();
     let travelTime2 =
       trainrunSection1.getSourceArrivalConsecutiveTime() -
       trainrunSection1.getTargetDepartureConsecutiveTime();
     travelTime1 = travelTime1 < 0 ? travelTime2 : travelTime1;
     travelTime2 = travelTime2 < 0 ? travelTime1 : travelTime2;
     const calculatedTravelTime = Math.min(travelTime1, travelTime2);
     const halteZeit = Math.min(minHalteZeitFromNode, Math.max(0, calculatedTravelTime - 2));
-    const travelTimeIssue = travelTime1 === travelTime2 || minHalteZeitFromNode !== halteZeit;
+    const travelTimeIssue = travelTime1 !== travelTime2 || minHalteZeitFromNode !== halteZeit;
```

Comment:
The code is unecessary dense here. My suggestion (remove the above lines also):

```suggestion
    const calculatedTravelTime = Math.abs(
      trainrunSection1.getTargetArrivalConsecutiveTime() -
        trainrunSection1.getSourceDepartureConsecutiveTime(),
    );
    const halteZeit = Math.min(minHalteZeitFromNode, Math.max(0, calculatedTravelTime - 2));
    const travelTimeIssue = !calculatedTravelTime || minHalteZeitFromNode !== halteZeit;
```